logtail.js

```diff
@@ -53,21 +57,23 @@ function get_log() {
                 if (!c_r)
                     throw "Server did not respond with a Content-Range";
 
-                size = parseInt(c_r.split("/")[1]);
-                if (isNaN(size))
+                log_file_size = parseInt(c_r.split("/")[1]);
+                content_size = xhr.getResponseHeader("Content-Length");
+                
+                if (isNaN(log_file_size))
                     throw "Invalid Content-Range size";
             } else if (xhr.status === 200) {
-                if (log_size > 1)
+                if (!first_load)
                     throw "Expected 206 Partial Content";
 
```

Comment:
If the file is truncated, the server will send us `416: Requested range not satisfiable`, which is dealt with elsewhere. It shouldn't send us a 200.

I think cherokee might have replied to requests with `Range: bytes=0-` or `Range: bytes=-NNN` where `NNN` is larger than the file (i.e., the Range header covered the entire file) with a 200.

Nginx does not. To that extent, we /always/ expect a 206 Partial Content, at the moment. I'll remove this branch.